### DISCUSSION TITLE
Download Package Installer

### DISCUSSION THREAD

**Discussion by georgeinva2004** [basicx-StrgV/WGet.NET#30] [General]
Great library!

The documentation page shows "Download Package Installer" as provided functionality.  Two questions:  I'm not clear on what it does.  Will it download and install WinGet?  If so, which class/method has this functionality?

Thanks in advance.

-John

**Comment by basicx-StrgV** [comment]
Hi,  
the function can be found in the ‘WinGetPackageManager’ class and it will download the installer for the provided package (e.g. 7zip or Notepad++) to the provided directory.  

Here is a link to the documentation: https://basicx-strgv.github.io/WGet.NET/d1/d84/a00097.html#a7c7c853fee8cf0545cf6bb22c918be7e

Thanks for informing me that this point is not clear in the readme. I will see what I can do to improve it.

**Comment by georgeinva2004** [comment]
Thank-you so much!!!!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
